test/csuite/incr_backup/main.c

```diff
@@ -822,15 +564,14 @@ main(int argc, char *argv[])
     uint32_t file_max, iter, max_value_size, next_checkpoint, rough_size, slot;
     int ch, ncheckpoints, nreopens;
     const char *backup_verbose, *working_dir;
-    char backup_check[1024], backup_dir[1024], backup_src[1024], conf[1024], home[1024];
+    char backup_src[1024], conf[1024], home[1024];
```

Comment:
Instead of identifying the directories by the function comment, you could identify the directories by declaring each dir name variable on a separate line with a comment giving the purpose of that directory. That is probably better for main. I used the function comment for the other functions since that makes the diretories part of the function interface.

` char backup_src[1024]; /* A copy of the database used to check after reopening the connection. */`

Reply:
Our coding guidelines don't have us breaking each variable onto a separate line.

Reply:
I think changing guidelines would be a much more general ticket to change it all over the code base. My change simplifies this a lot by removing some of them.